src/azure-cli/azure/cli/command_modules/netappfiles/_help.py

```diff
@@ -330,7 +330,7 @@
     short-summary: The service level
   - name: --usage-threshold
     short-summary: The maximum storage quota allowed for a file system as integer number of GiB. Min 100 GiB, max 100TiB"
-  - name: --creation-token
+  - name: --file-path
```

Comment:
For your command `netappfiles volume create`, a parameter is renamed, right? This is a breaking change for users.  Typically, for breaking changes, we need to emit out a warning to tell users for 2 months. But given your commands only involve 16 subscriptions in  the past 30 days. You can just make the change and call out your change  in History.rst as follows:
[BREAKING CHANGE] netappfiles volume create: Rename parameter --creation-token to --file-path

Reply:
Ok I will update the history. However, since is this NetApp CLI really subject to breaking change rules given that it explicitly sets 'is_preview=True' in the commands.py?

Reply:
Good point! If no preview flag, you cannot do such kind of breaking change immediately. For stable commands, you need to show up breaking changes message for two sprints and then you can do the change.

Reply:
ok will update the history